ethereum/testutils.py

```diff
@@ -575,6 +577,24 @@ def fixture_to_bytes(value):
         return value
 
 
+def get_config_overrides(filename):
+    override = {}
+    parts = filename.split(os.sep)
+    if 'Homestead' in parts:
+        override['HOMESTEAD_FORK_BLKNUM'] = 0
+    if 'TestNetwork' in parts:
```

Comment:
Are only 'TestNetwork' and 'EIP150' mutually exclusive? Shouldn't the `if` `elif` chain also encompass the `TestNetwork`?